src/views/studio/VideoWorkspace/VideoWorkspace.style.ts

```diff
@@ -14,14 +39,27 @@ export const Container = styled(animated.div)`
   flex-direction: column;
   background-color: ${oldColors.gray[900]};
   box-shadow: 0 4px 52px ${oldColors.black};
-`
+  opacity: 1;
+  transform: translateY(0);
+  transition: top ${cVar('animationTransitionSlow')};
 
-export const DrawerOverlay = styled(animated.div)`
-  position: fixed;
-  z-index: ${zIndex.videoWorkspaceOverlay};
-  top: 0;
-  right: 0;
-  bottom: 0;
-  left: 0;
-  background-color: ${oldColors.transparentBlack[54]};
+  &.video-workspace {
+    &-exit,
+    &-enter-active,
+    &-enter-done {
+      opacity: 1;
+      transform: translateY(0);
+      transition: all ${cVar('animationTransitionSlow')};
+    }
+
+    &-enter,
+    &-exit-active {
+      transform: translateY(100%);
+      opacity: 0;
+    }
+
+    &--minimized {
+      top: calc(100% - ${VIDEO_WORKSPACE_TABS_BAR_HEIGHT}px - 1px);
```

Comment:
Where does that `1px` come from? From my quick testing it seems it works better without it

Reply:
Also, why are we using `top` here instead of `transform`? This will make the animation less performant

Reply:
TBH, I took this `1px` from previous implementation https://github.com/Joystream/atlas/pull/1652/files#diff-169352e72b00dd3da5f910131bf7694dd34eaa84124ea8a38c49b8594a6c1cd9L13